spec/support/view_component/matchers.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Support
+  module ViewComponent
+    module Matchers
+      extend RSpec::Matchers::DSL
+
+      define :have_rendered_component do |component_class|
+        match do |rendered|
+          @stubs = _test_context.components.select { |stubbed| stubbed.instance.is_a? component_class }
+          @matching = @stubs.select { |stubbed| arguments.args_match?(*stubbed.args) }
+          @matching.any? do |stubbed|
+            rendered.content.include?(stubbed.output)
```

Comment:
Is this checking that the correct lorem text is present in the parent?

Reply:
Yes :) This will catch errors like having <% instead of <%=